proto/osmosis/concentratedliquidity/v1beta1/genesis.proto

```diff
@@ -6,10 +6,10 @@ import "cosmos_proto/cosmos.proto";
 import "cosmos/base/v1beta1/coin.proto";
 import "google/protobuf/any.proto";
 import "osmosis/accum/v1beta1/accum.proto";
-import "osmosis/concentrated-liquidity/params.proto";
-import "osmosis/concentrated-liquidity/position.proto";
-import "osmosis/concentrated-liquidity/tickInfo.proto";
-import "osmosis/concentrated-liquidity/incentive_record.proto";
+import "osmosis/concentratedliquidity/params.proto";
+import "osmosis/concentratedliquidity/v1beta1/position.proto";
+import "osmosis/concentratedliquidity/v1beta1/tickInfo.proto";
+import "osmosis/concentratedliquidity/v1beta1/incentive_record.proto";
 
```

Comment:
i hope this isnt a break

Reply:
explained on call, this is required due to proto paths now needing to match folder paths. If we want to retain the old import paths, we need to change the proto paths which will break integrators.